### DISCUSSION TITLE
New student

### DISCUSSION THREAD

**Discussion by Gmore1122** [community/community#48277] [GitHub Education]
Hi Guys I can’t believe am speaking with almost everyone across the globe 
Am Thomas chatting from Nigeria and also a new student happy to meet you all

**Comment by Miniontoby** [comment]
I am also a new student. I am from the Netherlands.

Nice to meet you!

**Comment by Gmore1122** [comment]
Am pleased meeting you too brother 
But how do I know about this environment

**Comment by melvincwng** [comment]
Hey @Gmore1122 and @Miniontoby 👋, nice to e-meet both of you all!

Welcome to GitHub Community! Feel free to ask in this discussion forums any technical or non-technical questions related to GitHub/programming.

For starters, you can consider introducing yourself in the [Introduction thread](https://github.com/orgs/community/discussions/23204).

To learn more about GitHub, you can use this course right here - [GitHub Skills](https://skills.github.com/).

Take care, and have a good day ahead 😃
